tensorflow/tools/gcs_test/Dockerfile

```diff
@@ -14,6 +14,10 @@ RUN curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/install_google_clou
 RUN chmod +x install_google_cloud_sdk.bash
 RUN ./install_google_cloud_sdk.bash --disable-prompts --install-dir=/var/gcloud
 
+RUN curl -O https://bootstrap.pypa.io/get-pip.py && \
```

Comment:
This change was initially there to make sure that the pip install of tensorflow works. There are some recent changes (probably the numpy version update) which caused this to stop working. But I realized that the problem can be more easily solved with using ubuntu 16 instead of 14. See new changes.